plugin-api-agent/src/commonMain/kotlin/com/epam/drill/plugin/api/processing/AgentPart.kt

```diff
@@ -38,9 +38,6 @@ abstract class AgentPart<A>(
     //TODO remove from API - this is only used in agent
     open fun load(on: Boolean) {
         initPlugin()
-        if (on) {
```

Comment:
reason for removing `on()` call

Reply:
The load method is used only with the argument "on" equal to true. But that was not the reason for the removal. In fact, the real reason is no longer relevant, and it is possible to return this block of code, or remove the "load" method, as indicated in the comment above.